raiden/api/rest.py

```diff
@@ -1432,3 +1435,7 @@ def get_status(self) -> Response:
                 return api_response(result=dict(status="syncing", blocks_to_sync=to_sync))
             else:
                 return api_response(result=dict(status="unavailable"))
+
+    def shutdown(self) -> Response:
+        spawn_named("trigger shutdown", self.raiden_api.shutdown)
```

Comment:
I forgot, this has to be linked to the main greenlet, otherwise exceptions will be lost